src/kirby/components/loading-overlay/loading-overlay.component.tns.html

```diff
@@ -0,0 +1,18 @@
+<GridLayout #overlayWrapperRef>
+    <StackLayout class="content-wrapper">
+        <ng-content></ng-content>
+    </StackLayout>
+    <FlexboxLayout
+    *ngIf="isLoading"
```

Comment:
I can't find the article again now, but i read somewhere that the use of `*ngIf` can give issues in {N}.
Instead we should use `[visibility]="isLoading ? 'visible' : 'collapse'`.

I will try to find the article again, but i am a bit unsure about it. Maybe someone else can remember it?

Reply:
You are right, but this is outdated now :)